src/lister_haxe/interpret.sh

```diff
@@ -0,0 +1 @@
+haxe -main $1 --interp
```

Comment:
I could have made this not take a parameter, which would also have the side effect of having a single file that could work as a shell script for OSX/Linux/Windows (I think), but I wanted the flexibility of being able to run different classes as the main entry point.